lib/pos_blockchain.go

```diff
@@ -1602,7 +1617,7 @@ func (bc *Blockchain) removeTipBlockFromBestChain() *BlockNode {
 // runCommitRuleOnBestChain commits the grandparent of the block if possible.
 // Specifically, this updates the CommittedBlockStatus of its grandparent
 // and flushes the view after connecting the grandparent block to the DB.
-func (bc *Blockchain) runCommitRuleOnBestChain() error {
+func (bc *Blockchain) runCommitRuleOnBestChain(verifySignatures bool) error {
```

Comment:
Just for the sake of discussion. Can we live without adding the flag here? If a bad block is committed, the node would have to restart and resync (probably from the previous HyperSync epoch)

Reply:
we *could* but it will definitely slow things down when syncing. Signature verification of txns when connect a block can add significant time to the sync. If we got to the point where we're running the commit rule, we should feel pretty confident about the the block that is being committed.